### PR TITLE
PC-186: Improve Similar Cases table UI and rename Disorders column

### PR DESCRIPTION
**Pull request by katpavlova**
Authored with @veronikaslc

### CODE REVIEW

**Comment by katpavlova** [comment]
@sashaandjic Please test this and let me know if you are satisfied with the UI or if anything needs tweaking.

**Comment by sashaandjic** [comment]
Diagnosis column is now too narrow. I would at least double the current width. Otherwise everything looks fantastic!
screen shot 2017-06-13 at 11 32 27 am

**Comment by marta-** [comment]
@sashaandjic The column with self-adjusts to the content (relatively to the other columns). I think it's fine.

I don't like the arrows. We should be consistent with the Show/Hide buttons for form sections and use [+] [-] icons instead.

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 1]
```diff
@@ -613,8 +613,8 @@
 
       // Provide access to the actual matching details
 
-      var showDetails = new Element("a", {"href" : "#", "class" : "button secondary tool show"}).update("$escapetool.javascript($services.localization.render('phenotips.similarCases.details.show'))");
-      var hideDetails = new Element("a", {"href" : "#", "class" : "button secondary tool hide"}).update("$escapetool.javascript($services.localization.render('phenotips.similarCases.details.hide'))");
+      var showDetails = new Element("a", {"href" : "#", "class" : "button secondary tool show collapsed", "id" : "similarity-button"}).update("$escapetool.javascript($services.localization.render('phenotips.similarCases.details.show'))");
+      var hideDetails = new Element("a", {"href" : "#", "class" : "button secondary tool hide", "id" : "similarity-button"}).update("$escapetool.javascript($services.localization.render('phenotips.similarCases.details.hide'))");
```
Comment (sdumitriu) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: It's illegal to have two elements with the same `id`.

Comment (sdumitriu) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Actually, are these buttons for every similar patient?

Comment (sashaandjic) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Add "(OMIM)" to "Diagnosis" in the header.

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 2]
```diff
@@ -628,12 +628,14 @@
         event.stop();
         showDetails.hide();
         hideDetails.show();
+        $('similarity-button').removeClassName('collapsed');
```
Comment (sdumitriu) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: This code does nothing useful actually, since you're uncollapsing the button that is now hidden.

**Comment by katpavlova** [comment]
@sdumitriu @sashaandjic @marta- The requested changes have been implemented, please review & merge.

**Comment by sashaandjic** [comment]
tested. ui is looking really good.
screen shot 2017-06-14 at 11 15 28 pm

**Pull request merged**